cartofacade/carto_facade_test.go

```diff
@@ -106,7 +106,7 @@ func TestRequest(t *testing.T) {
 		test.That(t, err, test.ShouldResemble, expectedErr)
 	})
 
-	t.Run("test requesting with when the work function takes longer than the timeout", func(t *testing.T) {
+	t.Run("fails when the work function takes longer than the timeout", func(t *testing.T) {
```

Comment:
[opt]
```suggestion
	t.Run("failure due to timeout", func(t *testing.T) {
```

Reply:
opting out to opt in for Esha's preference :D